src/App/JwtHandler.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+namespace Source\App;
+
+use Firebase\JWT\JWT;
+use Firebase\JWT\Key;
+
+class JwtHandler
+{
+    public static function makeJWT(int $userId): string
+    {
+        $secretKey = Registry::get('secretKey');
+        $date = new \DateTimeImmutable();
+        $expireAt = $date->modify('+10 minutes')->getTimestamp();
+        $domainName = Registry::get('domain');
+        $alg = Registry::get('jwtAlg');
+        $requestData = [
+            'iat' => $date->getTimestamp(),         // Issued at: time when the token was generated
+            'iss' => $domainName,                       // Issuer
+            'nbf' => $date->getTimestamp(),         // Not before
+            'exp' => $expireAt,                           // Expire

```

Comment:
лишние переменные особо не нужны
```suggestion
            'exp' => $date->modify('+10 minutes')->getTimestamp(),                           // Expire

```

Reply:
+